### PR TITLE
Boston2018

### PR DESCRIPTION
**Pull request by AndrewItIs**
pull request to have a solid master and a base master.

### CODE REVIEW

File: src/org/usfirst/frc/team467/robot/Logging.java [hunk 1]
```diff
@@ -16,48 +16,54 @@
...
-//		Logger.getLogger(RobotShape.class).setLevel(Level.WARN);
-//		Logger.getLogger(Rumbler.class).setLevel(Level.WARN);
-//		Logger.getLogger(TalonSpeedControllerGroup.class).setLevel(Level.INFO);
-//		Logger.getLogger(VisionIntegration.class).setLevel(Level.WARN);
-//		Logger.getLogger(XBoxJoystick467.class).setLevel(Level.WARN);
-//		Logger.getLogger(TiltMonitor.class).setLevel(Level.INFO);
-	}
 
+		config.addLogger(Action.class.getName(), new LoggerConfig(Action.class.getName(), Level.INFO, true));
+		config.addLogger(Drive.class.getName(), new LoggerConfig(Drive.class.getName(), Level.INFO, true));
```
Comment (Matthew-En) [inline src/org/usfirst/frc/team467/robot/Logging.java]: Turn most of the configs in Logging into comments, it's a little too much.

File: src/org/usfirst/frc/team467/robot/Ramp.java [hunk 2]
```diff
@@ -45,15 +46,15 @@ public State getState() {
 	public void lift() {
 		if (state == State.DOWN) {
 			solenoid.set(DoubleSolenoid.Value.kForward);
-			LOGGER.info(name + " lifting");
+			LOGGER.info("Lifting = {}", name);
```
Comment (Matthew-En) [inline src/org/usfirst/frc/team467/robot/Ramp.java]: This should be {"{} lifting", name);

File: src/org/usfirst/frc/team467/robot/Drive.java [hunk 3]
```diff
@@ -318,19 +316,20 @@ public double absoluteDistanceMoved() {
...
-		LOGGER.debug("The absolute distance moved: " + lowestAbsDist);
+		LOGGER.debug("The absolute distance moved: {}", lowestAbsDist);
 		return lowestAbsDist;
 	}
 
 	private double feetToTicks(double feet) {
 		double ticks = (feet / (RobotMap.WHEEL_CIRCUMFERENCE / 12.0)) * RobotMap.WHEEL_ENCODER_CODES_PER_REVOLUTION;
-		LOGGER.trace(feet + " feet = " + ticks + " ticks.");
+		LOGGER.trace("Feet = {} ticks = {}", feet, ticks);
+		//what do i do here
```
Comment (Matthew-En) [inline src/org/usfirst/frc/team467/robot/Drive.java:326]: Please remove this comment, it's not needed.

**Review by mahirsowad3 — APPROVED** [review]
I think that these changes are good for the most part if they worked in Boston. They can be merged into master.

**Review by Matthew-En — APPROVED** [review]

**Pull request merged**